data/src/languages_update.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
```

Comment:
Coudl you please remove all in-function whitespace? It makes it hard to parse when the function ends. If you really want to chunk it somehow, add a comment instead.